### PR TITLE
[stable-5] Add github workflow

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Add github workflow

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
I mistakenly applied `mergeit`, but removed it, so, I assume it won't get merged.

File: .github/workflows/ansible-bot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: ansible bot
```
Comment (abikouo) [inline .github/workflows/ansible-bot.yml:1]: This is useless here, this can of workflow are defined on the default branch

**Review by abikouo — APPROVED** [review]

**Review by gravesm — APPROVED** [review]

**Pull request merged**